lsp4clj/.gitignore

```
@@ -0,0 +1,5 @@
+*.jar
+*.class
+/.cpcache
+/.clj-kondo
+/.lsp
```

The only LSP/clj-kondo folders that always should be included on gitignore are these
```suggestion
/.clj-kondo/.cache
/.lsp/.cache
```